Makefile

```diff
@@ -112,3 +112,11 @@ dev-quick-install:
 	@make load-fixtures
 	echo "Creating superuser"
 	@make create-super-user
+
+shell:
+	clear
+	docker exec -it sadilar-terminology-web bash
```

Comment:
There is already a make target called shell.

Reply:
thank you for picking that up